bin/develop.js

```diff
@@ -0,0 +1,22 @@
+import program from 'commander'
```

Comment:
Ah indeed, I'll will adjust this to utilize requires instead along with ES5 compliant code.